public/js/reading/ReadingMain.js

```diff
@@ -0,0 +1,51 @@
+import {Library} from './components/Library.js';
+import {Text} from './components/Text.js';
+/**
+ * Represent Home scene
+ */
+
+export class ReadingMain{
```

Comment:
nit: add space between class name and curly bracket `export class ReadingMain {`